test/integration/kubernetes_deploy_test.rb

```diff
@@ -446,6 +446,25 @@ def test_deployment_with_progress_times_out_for_short_duration
     ])
   end
 
+  def test_deployment_with_timeout_override
+    result = deploy_fixtures("long-running", subset: ['undying-deployment.yml.erb']) do |fixtures|
+      deployment = fixtures['undying-deployment.yml.erb']['Deployment'].first
+      deployment['spec']['progressDeadlineSeconds'] = 5
+      deployment["metadata"]["annotations"] = {
+        KubernetesDeploy::KubernetesResource::TIMEOUT_OVERRIDE_ANNOTATION => "10S",
+      }
+      container = deployment['spec']['template']['spec']['containers'].first
+      container['readinessProbe'] = { "exec" => { "command" => ['- ls'] } }
+    end
+    assert_deploy_failure(result, :timed_out)
+
+    assert_logs_match_all([
```

Comment:
`KubernetesDeploy::KubernetesResource::STANDARD_TIMEOUT_MESSAGE` instead of copying and pasting.

Reply:
Unfortunately that doesn't appear to work due to line splitting in the logs :(

Reply:
Ah, nm I figured out why it wasn't working before